### DISCUSSION TITLE
psrule for Cloud-scale analytics?

### DISCUSSION THREAD

**Discussion by ievsantillan** [microsoft/PSRule.Rules.CAF#111] [Ideas]
psrule for [Cloud-scale analytics](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/cloud-scale-analytics/)?

**Comment by ElinoraWhite** [comment]
@isdataninja Are you able to expand on your idea so we can understand what this might look like in your ideal state? Thanks.
